src/platform/packages/shared/kbn-unified-metrics-grid/src/components/toolbar/dimensions_selector.tsx

```diff
@@ -117,6 +128,42 @@ export const DimensionsSelector = ({
     );
   }, [selectedDimensions]);
 
+  const clearAllSection = useMemo(() => {
```

Comment:
We're repeating this component here and in the values_selector. Should we extract it into a new file and reuse it on both components?

Reply:
Good idea! I expected that they would have more differences when I added them. I created a component and reused it 👍